app/templates/components/orders/order-summary.hbs

```diff
@@ -65,12 +73,16 @@
           <th></th>
           <th>
             <div class="ui aligned small primary icon">
-              {{t 'Order Total'}}:
+              {{t 'Grand Total'}}:
             </div>
           </th>
           <th colspan="4">
             <div class="ui aligned small primary icon">
-              {{currency-symbol eventCurrency}} {{number-format data.amount}}
+              {{#if event.tax.isTaxIncludedInPrice}}
+                {{currency-symbol eventCurrency}} {{number-format data.amount}}
+              {{else}}
+                {{currency-symbol eventCurrency}} {{number-format taxed_price}}
+              {{/if}}
```

Comment:
No change here. `{{currency-symbol eventCurrency}} {{number-format data.amount}}` Is sufficient now. :)